metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -196,11 +196,25 @@ def folder_ancestors(
         fields: Union[str, List[str]] = ["id", "name", "parent_id"],
     ) -> Sequence[Folder]:
         self.client_stats.folder_calls += 1
-        return self.client.folder_ancestors(
-            folder_id,
-            self.__fields_mapper(fields),
-            transport_options=self.transport_options,
-        )
+        try:
+            return self.client.folder_ancestors(
+                folder_id,
+                self.__fields_mapper(fields),
+                transport_options=self.transport_options,
+            )
+        except SDKError as e:
+            if "Looker Not Found (404)" in str(e):
+                # User not found
+                logger.info(
+                    f"Could not find ancestors for folder with id {folder_id}: 404 error"
+                )
+            else:
+                logger.warning(
+                    f"Could not find ancestors for folder with id {folder_id}"
+                )
```

Comment:
should we have an info-level thing added to the report? or is this ignorable?

Reply:
Yes, unfortunately report is not yet present here in lib_wrapper.
Replicated the handling of user not found.

Per looker -> this could be permission error -> https://cloud.google.com/looker/docs/api-troubleshooting#not_found_404_errors
So ideally we could report them as info or warning in report. Will followup with report related changes when I get a chance.